docs/dev/reference/dca/callbacks.md

```diff
@@ -626,7 +626,7 @@ class ExampleListOperationListener
     )
     {}
 
-    public function onListCustomOperationCallback(
+    public function onButtonCallback(
```

Comment:
```suggestion
    public function __invoke(
```

Let's use `__invoke` instead so that we don't have to worry about naming the function.